rekcurd.proto

```diff
@@ -47,7 +49,7 @@ enum EnumVersionInfo {
 // RekcurdMessage message.
 message RekcurdMessage {
     Status status = 1;
-    JsonFormatString meta = 2;
+    google.protobuf.Value meta = 2;
```

Comment:
Use [this Seldon spec](https://github.com/SeldonIO/seldon-core/commit/01126990f038d95f22a6ea1594c7a93666c42225#diff-259f06121c12bcfcc450830e854ef547)